frontend/.eslintrc.json

```diff
@@ -19,6 +19,13 @@
     "plugins": [
         "react"
     ],
+    "settings": {
+        "import/resolver": {
+          "node": {
+            "extensions": [".js", ".jsx", ".ts", ".tsx"]
+          }
+        }
+    },
```

Comment:
What does this do

Reply:
@avidlaud The below error pops up after pulling from the main branch and trying to run the project even after running npm install for axios and react-chartjs-2, so that code helps to fix the problem. 

<img width="665" alt="axios" src="https://user-images.githubusercontent.com/52171753/112528734-e0aea300-8d7a-11eb-9848-9286ade9ca35.png">

Reply:
Gotcha